graph/cytoscape/cytoscape.go

```diff
@@ -30,17 +31,18 @@ type NodeData struct {
 	Parent string `json:"parent,omitempty"` // Compound Node parent ID
 
 	// App Fields (not required by Cytoscape)
-	Service          string `json:"service"`
-	Version          string `json:"version,omitempty"`
-	Rate             string `json:"rate,omitempty"`             // edge aggregate
-	Rate3xx          string `json:"rate3XX,omitempty"`          // edge aggregate
-	Rate4xx          string `json:"rate4XX,omitempty"`          // edge aggregate
-	Rate5xx          string `json:"rate5XX,omitempty"`          // edge aggregate
-	IsCircuitBreaker string `json:"isCircuitBreaker,omitempty"` // true | false | unknown
-	IsGroup          string `json:"isGroup,omitempty"`          // set to the grouping type, current values: [ 'version' ]
-	IsRoot           string `json:"isRoot,omitempty"`           // true | false
-	IsSelfInvoke     string `json:"isSelfInvoke,omitempty"`     // rate of selfInvoke
-	IsUnused         string `json:"isUnused,omitempty"`         // true | false
+	Service             string                     `json:"service"`
+	Version             string                     `json:"version,omitempty"`
+	Rate                string                     `json:"rate,omitempty"`             // edge aggregate
+	Rate3xx             string                     `json:"rate3XX,omitempty"`          // edge aggregate
+	Rate4xx             string                     `json:"rate4XX,omitempty"`          // edge aggregate
+	Rate5xx             string                     `json:"rate5XX,omitempty"`          // edge aggregate
+	IsCircuitBreaker    string                     `json:"isCircuitBreaker,omitempty"` // true | false | unknown
+	IsGroup             string                     `json:"isGroup,omitempty"`          // set to the grouping type, current values: [ 'version' ]
+	IsRoot              string                     `json:"isRoot,omitempty"`           // true | false
+	IsSelfInvoke        string                     `json:"isSelfInvoke,omitempty"`     // rate of selfInvoke
+	IsUnused            string                     `json:"isUnused,omitempty"`         // true | false
+	DestinationPolicies models.DestinationPolicies `json:"destinationPolicies,omitempty"`
```

Comment:
We should not be putting the entire set of destination policies in the graph. Shouldn't this JSON be restricted to only what the graph needs? If the client needs more, can't it call some other APIs? Seems to me this is going to increase the size of the graph and potentially slow up the retrieval of the graph JSON. Perhaps we need some numbers on efficiency of the graph generation API now that this is added. Just seems like we don't need ALL of that (e.g. for the circuit breaker badge, we really only need to know THAT a circuit breaker exists, we don't care about anything about the CB, just that it exists).